spec/lib/sufia/user_stat_importer_spec.rb

```diff
@@ -10,6 +10,7 @@
 
   let(:bilbo) { FactoryGirl.create(:user, email: 'bilbo@example.com') }
   let(:frodo) { FactoryGirl.create(:user, email: 'frodo@example.com') }
+  let!(:gollum) { FactoryGirl.create(:user, email: 'gollum@example.com') }
```

Comment:
Since gollum is only used in the expect condition (gollum.id) I had to use `let!` to force RSpec to execute the statement. Otherwise the record wouldn't be created. I spent like 20 minutes scratching my head on why the record was there.